test/lib/lbt/analyzer/XMLTemplateAnalyzer.js

```diff
@@ -33,6 +33,31 @@ test("integration: Analysis of an xml view", async (t) => {
 		"Implicit dependency should be added since an XMLView is analyzed");
 });
 
+test("integration: Analysis of an xml view with data binding in properties", async (t) => {
+	const xml = `<mvc:View xmlns:mvc="sap.ui.core.mvc" xmlns:core="sap.ui.core"
+		controllerName="myController">
+			<core:ComponentContainer async="true" name="{/component}" />
+		</mvc:View>`;
+	const mockPool = {async findResource(name) {
+		return {
+			buffer: () => name.endsWith(".xml") ? JSON.stringify(xml): "test"
```

Comment:
JSON.stringify?

Reply:
Copy & past from existing tests...
But yes, this doesn't really make sense. The function is not called at all.
Maybe @tobiasso85 can comment on this.